pipeline_dp/contribution_bounders.py

```diff
@@ -192,31 +186,32 @@ def rekey_per_privacy_id_per_partition_key(pid_pk_v_values):
                 yield (privacy_id, partition_key), values
 
         col = backend.flat_map(col, rekey_per_privacy_id_per_partition_key,
-                               "Unnest per privacy id")
+                               "Unnest per privacy_id")
         # ((privacy_id, partition_key), [value])
 
         return backend.map_values(
             col, aggregate_fn,
-            "Apply aggregate_fn after per privacy id contribution bounding")
+            "Apply aggregate_fn after cross-partition contribution bounding")
 
 
 def collect_values_per_partition_key_per_privacy_id(
         col, backend: pipeline_backend.PipelineBackend):
-    """Groups per privacy id and collects values per partition key.
+    """Collects values per partition key in grouped by privacy_id collection.
```

Comment:
nit: Not sure what this means. `Collects values per partition key keyed by privacy_id`?

Reply:
Updated docstring PTAL